bolt/database.rules.bolt

```diff
@@ -0,0 +1,113 @@
+type Event {
```

Comment:
Personally I would move these into the `database.types.bolt` but its up to the rest of the team - thoughts?

Reply:
Go for it